### PR TITLE
fix: fixed avatar response

### PR DESCRIPTION
**Pull request by DavidKorystov**

### CODE REVIEW

File: src/repositories/user.repository.ts [hunk 1]
```diff
@@ -47,7 +47,8 @@ export const updateOne = async (
       password: data.password && hashPassword(data.password),
       profile: (data.profile || file) && {
         ...data.profile,
-        avatar: file && file?.buffer.toString("base64"),
+        avatar:
+          file && `data:image/png;base64,${file?.buffer.toString("base64")}`,
```
Comment (VitaliiDenysiuk) [inline src/repositories/user.repository.ts]: Why did you do that? Why "data:image/png;base64"? why "png"?

**Review by VitaliiDenysiuk — CHANGES_REQUESTED** [review]
Why did you do that?

File: src/repositories/user.repository.ts (hunk 1 above)
Comment (DavidKorystov) [reply src/repositories/user.repository.ts]: @VitaliiDenysyuk can we somehow get the format of the incoming avatar to apply to this line?
or is this enough: "data:image;base64" ?

Comment (VitaliiDenysiuk) [reply src/repositories/user.repository.ts]: @DavidKorystov,  I did not analyze all logic but it is very strange when you take something from database and add some string. And how do you know what type data in it?

Comment (alexandrtovmach) [reply src/repositories/user.repository.ts]: `file` should contain information about mime type

Comment (alexandrtovmach) [reply src/repositories/user.repository.ts]: something like `data:image/${file.mimetype};base64,${file?.buffer.toString("base64")}`

File: src/repositories/user.repository.ts [hunk 2]
```diff
@@ -47,11 +47,16 @@ export const updateOne = async (
...
       profile: (data.profile || file) && {
         ...data.profile,
-        avatar: file && file?.buffer.toString("base64"),
+        avatar:
+          file &&
+          `data:${file.mimetype};base64,${file?.buffer.toString("base64")}`,
       },
     }
   );
+  console.warn(file);
```
Comment (DavidKorystov) [inline src/repositories/user.repository.ts]:
```suggestion
```

**Review by VitaliiDenysiuk — APPROVED** [review]
lgtm

**Review by alexandrtovmach — APPROVED** [review]

**Pull request merged**